src/Services/request.ts

```diff
@@ -0,0 +1,62 @@
+import { soxa } from "../../deps.ts";
+import {
+  EServiceKindError,
+  GithubErrorResponse,
+  GithubExceedError,
+  QueryDefaultResponse,
+  ServiceError,
+} from "../Types/index.ts";
+
+export async function requestGithubData<T = unknown>(
+  query: string,
+  variables: { [key: string]: string },
+  token = "",
+) {
+  const response = await soxa.post("", {}, {
+    data: { query, variables },
+    headers: {
+      Authorization: `bearer ${token}`,
+    },
+  }) as QueryDefaultResponse<{ user: T }>;
+  const responseData = response.data;
+
+  if (responseData.data.user) {
```

Comment:
The following errors may occur?
`Cannot read property 'user' of undefined`

Reply:
yeah, type safety, thanks